### PR TITLE
OpenCensus Samples Folder + first sample for Azure Monitor Exporter + update image for circleci

### PR DESCRIPTION
**Pull request by lzchen**
This PR creates a `opencensus-samples` folder that will hold various sample applications for different exporters and integrations.

The sample in this PR is the first sample for Azure Monitor. It is a `flask` application that communicates with a small `sqlite` database using `sqlalchemy` APIs in the form of a simple "to-do" app. Refer to the README on more details.

We will expect users to install the sample packages individually per exporter, and not the entire `opencensus-samples` package.

image

### CODE REVIEW

File: contrib/opencensus-samples/azure_monitor/src/azure_monitor/flask_sample/app/__init__.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Copyright 2019, OpenCensus Authors
```
Comment (hectorhdzg) [inline contrib/opencensus-samples/azure_monitor/src/azure_monitor/flask_sample/app/__init__.py]: Maybe having the sample inside https://github.com/census-instrumentation/opencensus-python/tree/master/contrib/opencensus-ext-azure/examples would be cleaner, having this other folder just make it harder to discover

File: contrib/opencensus-samples/azure_monitor/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Install OpenCensus Azure Monitor samples through `pip`.
+
+`pip install opencensus-azure-monitor-samples`
+
+
+Flask Sample
+------------
+
+The Azure Monitor Flask sample is a simple "To-Do" application.
```
Comment (hectorhdzg) [inline contrib/opencensus-samples/azure_monitor/README.rst]: Why not having all description of the sample in a single paragraph?, the list format confused me a little bit

File: .circleci/config.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ jobs:
       ignore:
         - gh-pages
     docker:
-      - image: googleapis/nox:0.17.0
+      - image: googleapis/nox:latest
```
Comment (hectorhdzg) [inline .circleci/config.yml]: Is this relevant to the sample app?

File: contrib/opencensus-samples/azure_monitor/README.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+4. Hit your local endpoint (should be http://localhost:5000). This should open up a browser to the main page.
+5. On the newly opened page, you can add tasks via the textbox under `Add a new todo item:`. You can enter any text you want (cannot be blank).
+6. Click `Add Item` to add the task. The task will be added under `Incomplete Items`.
+7. Each task has a `Mark As Complete` button. Clicking it will move the task from incomplete to completed.
+8. You can also hit the `blacklist` url page to see a sample of a page that does not have telemetry being sent (http://localhost:5000/blacklist).
+
+Telemetry
+^^^^^^^^^
+
+There are various types of telemetry that are being sent in the sample application.
```
Comment (hectorhdzg) [inline contrib/opencensus-samples/azure_monitor/README.rst]: A link to Azure docs describing how to see these events would be nice

File: contrib/opencensus-samples/azure_monitor/src/azure_monitor/flask_sample/app/__init__.py (hunk 1 above)
Comment (lzchen) [reply contrib/opencensus-samples/azure_monitor/src/azure_monitor/flask_sample/app/__init__.py]: The point of the sample package is so that users can install it and have them install integration packages as needed for the end-to-end applications. There might be also sample applications in the future that are for other exporters, so a generic `opencensus-samples` folder should be used since it is not `azure` specific.

File: .circleci/config.yml (hunk 3 above)
Comment (lzchen) [reply .circleci/config.yml]: This was to fix a build error I was getting regarding the image not having `pip` in the virtual env. Upgrading to `latest` fixes the error. I will update the title of the PR to include this.

**Comment by lzchen** [comment]
Moving this sample to different repo.

**Comment by lzchen** [comment]
Re-opened [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/pull/1)
